libafl/src/observers/map.rs

```diff
@@ -32,6 +34,27 @@ where
         self.map().len()
     }
 
+    /// Count the set bytes in the map
+    fn count_bytes(&self) -> u64 {
+        let initial = self.initial();
+        let cnt = self.usable_count();
+        let mut res = 0;
+        for x in self.map()[0..cnt].iter() {
+            if *x != initial {
+                res += 1;
+            }
+        }
+        res
+    }
+
+    fn hash(&self) -> u64 {
+        let ptr = self.map().as_ptr() as *const u8;
+        let map_size = self.map().len() / core::mem::size_of::<T>();
+        unsafe {
+            return xxh3::xxh3_64(from_raw_parts(ptr, map_size));
```

Comment:
We used `ahash` at other places of the code.
Probably we should eventually decide which one to keep, I would favor `ahash` since the hashmap we use already uses it, @andreafioraldi what to do you think?